gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Tue Aug 08 15:57:04 PDT 2017
+#Tue Aug 15 12:10:52 PDT 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.0.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.0.2-bin.zip
```

Comment:
Oh, hm. This is what gets generated by running `./gradlew wrapper`. I think I may have manually updated the distro before, and incorrectly updated this URL. Either way, this is the diff that keeps showing up. I'll amend the commit message to reflect the actual change taking place.